dts/bindings/power/nxp,aon-wakeup-pin.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Some NXP SoC's have pins dedicated to generate a wakeup interrupt.
+
+compatible: "nxp,aon-wakeup-pin"
+
+include: base.yaml
+
+properties:
+  interrupts:
+    type: array
+    required: true
+    description: Interrupt to wakeup the SoC from low power modes.
+
+  wakeup-level:
+    type: int
```

Comment:
int enum feels awkward, maybe let's do either a boolean or a string enum

Reply:
I have changed this to a String.